tests/cases/tm2eth/scripts/fixture

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eux
+
+DOCKER=docker
+FIXTURES_DIR=./fixtures
```

Comment:
To determine FIXTURES_DIR, please use the same pattern as fixture in tm2tm.

Reply:
Fixed